kuka_lbr_iiwa_support/urdf/lbr_iiwa_14_r820.urdf

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from lbr_iiwa_14_r820.xacro         | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
+<robot name="kuka_lbr_iiwa_14_r820" xmlns:xacro="http://ros.org/wiki/xacro">
+  <!-- link list -->
+  <link name="base_link">
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://kuka_lbr_iiwa_support/meshes/irb_iiwa_14_r820/visual/base_link.dae"/>
```

Comment:
On 3-6-2015 18:25, Levi Armstrong wrote:
> The naming is actually KUKA LBR iiwa 14kg R820.

yes, but you have 'meshes/irb_iiwa_14_r820'.